src/components/Categories/Categories.js

```diff
@@ -0,0 +1,44 @@
+// Info : contains data related to Categories component
+
+// Imports
+import React from 'react'
+import { useState, useEffect } from 'react';
+import axios from "axios";
+import './Categories.css'
+
+// Function Categroies
+const Categories = () => {
+
+  // Initialize State
+  const[categories, setCategories] = useState([])
+
+  // Runs only once
+  useEffect(()=>{
+    
+      // Get data from Db
+      axios.get('api/categories')
+      .then(response => setCategories(response.data.categories)) // If success update state
+      .catch(err => console.log(err)) // If error
+    },[])
+
+  // Return 
```

Comment:
you can remove comments which are self-explantory

Reply:
Added these comments to keep a consistent comment structure. But will keep your suggestion in mind